global.json

```diff
@@ -6,14 +6,10 @@
     "dotnet": "7.0.100-preview.2.22114.1",
     "runtimes": {
       "dotnet": [
-        "2.1.30",
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"
       ],
       "dotnet/x86": [
```

Comment:
@wtgodbe may want to move this section above `"dotnet"` so eng/scripts/prepare-sourcebuild-globaljson.sh works as expected.

@MichaelSimons the alternative would be to stop using eng/scripts/prepare-sourcebuild-globaljson.sh. Will that work correctly now❔

Reply:
It looks to me like we will still need ` eng/scripts/prepare-sourcebuild-globaljson.sh` because of the `dotnet/x86` entry.